capy/src/main/java/com/jocmp/capy/accounts/Credentials.kt

```diff
@@ -8,16 +9,18 @@ interface Credentials {
     val username: String
     val secret: String
     val url: String
+    val clientCertAlias: String
     val source: Source
 
-    suspend fun verify(): Result<Credentials>
+    suspend fun verify(clientCertManager: ClientCertManager): Result<Credentials>
```

Comment:
instead of parameterizing `verify` here, can you add `clientCertManager` as a parameter to the `ReaderCredentials` constructor? That way the Feedbin manager doesn't need to handle it as a no-op.